code/modules/capture_the_flag/ctf_controller.dm

```diff
@@ -115,7 +115,7 @@
 		if(user.ckey in enemy_team_members)
 			to_chat(user, span_warning("No switching teams while the round is going!"))
 			return FALSE
-		if(friendly_team_members.len > enemy_team_members.len)
+		else if(friendly_team_members.len > enemy_team_members.len)
```

Comment:
I'm not sure this changes anything, since the above control statement already calls an early return, meaning that, if the condition `user.ckey in enemy_team_mebers` is true, this control statement won't run anyway.

Reply:
![image](https://github.com/tgstation/tgstation/assets/93882977/586a456f-f2f7-4cfc-a14e-455bfb21f1a7)
I managed to get this message as a Guest, so maybe it just doesn't matter for anyone with ckey,  as i can't get it when logged in.

Reply:
It's perhaps something else that sends both messages, then, tho technically guests shouldn't be playing the server.